.env

```diff
@@ -75,7 +75,7 @@ _APP_MAINTENANCE_RETENTION_CACHE=2592000
 _APP_MAINTENANCE_RETENTION_EXECUTION=1209600
 _APP_MAINTENANCE_RETENTION_ABUSE=86400
 _APP_MAINTENANCE_RETENTION_AUDIT=1209600
-_APP_USAGE_AGGREGATION_INTERVAL=5
+_APP_USAGE_AGGREGATION_INTERVAL=50000
```

Comment:
Regarding above, then this change goes from 5s to 50s.

Reply:
yeah, I used an env that was used to aggregate via influx
Updated it to 60 sec (Eldad suggested)
Once it works we can tweak the sec